src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -4615,21 +4615,26 @@ checkNullable:
                 value = ParseExpressionCore()
 
             ElseIf modifiers.Any AndAlso modifiers.Any(SyntaxKind.OptionalKeyword) Then
```

Comment:
Don't know. That how it is current checked, I've just made the body of it language version dependent.